snowflake/objects/database/deploy.sql

```diff
@@ -0,0 +1,17 @@
+/*
+  The current maximum allowed execution depth of EXECUTE IMMEDIATE FROM
+  statements is 5. Unfortunately, that makes a call stack which looks like:
+
+  depth
+  0      deploy.sql =>
+  1        databases/deploy.sql =>
+  2          databases/recover/deploy.sql =>
+  3            databases/recover/schemas/deploy.sql =>
+  4              databases/recover/schemas/parquet/deploy.sql =>
+  5                databases/recover/schemas/parquet/tables/deploy.sql =>
+  6                  databases/recover/schemas/parquet/tables/enrolled_participants.sql
+
+  not possible. To circumvent this issue, we omit the highest level of
```

Comment:
If I am understanding this correctly to put into more layman terms: 

Instead of executing all database deployments through a single high-level script, we deploy each database individually from the main script (deploy.sql) where from `deploy.sql`, we directly call the lower-level deployment scripts for each database. This approach avoids hitting the maximum depth limit.?

Reply:
That's correct.